docs/PUBLISHING.md

```diff
@@ -93,7 +93,7 @@ access resin.io's production downloads S3 bucket.
 Run the following command to publish a specific file:
 
 ```sh
-./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -t <production|snapshot>
+./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -p <product name>
```

Comment:
But in that case, should it also document the `-k` flag to `aws-s3.sh`?
Given the way the check-for-upgrade code works, I think it's _very_ important to ensure that snapshot builds don't accidentally end up in the 'release' S3 bucket?